manifests/init.pp

```diff
@@ -99,19 +102,23 @@
   Variant[Boolean[false], Pattern[/icmp(v6|x)? type .+|tcp reset/]] $reject_with = 'icmpx type port-unreachable',
   Variant[Boolean[false], Enum['mask']] $firewalld_enable = 'mask',
   Optional[Array[Pattern[/^(ip|ip6|inet)-[-a-zA-Z0-9_]+$/],1]] $noflush_tables = undef,
+  Stdlib::AbsolutePath $nftables_configuration_path = '/etc/sysconfig/nftables.conf',
```

Comment:
Unixpath is probably fine, I don't think there is nftables on Windows?

Reply:
It is linux specific, I will change it to Unixpath